modules/options-for-notebook-server-environments.adoc

```diff
@@ -334,17 +409,27 @@ endif::[]
 endif::[]
 
 ifdef::upstream[]
-| CUDA - RStudio Server
-| 2023.2 (Recommended)
+.2+| CUDA - RStudio Server
+| 2024.1 (Recommended)
```

Comment:
I should check what exactly we are referring to since the RStudio isn't a prebuilt image.

Reply:
I'd included these for reference because the CUDA version that the CUDA - RStudio image uses is updated for 2.9. Open to suggestions, should we include RStudio in this table at all?

Reply:
Raised with the team for feedback. 👍

Reply:
Confirmed, we'll show only the 2024.1 version in 2.9 for the RStudio images.